docs/Advanced-Topics.md

```diff
@@ -62,12 +62,18 @@ you created it
 +---------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | `judges_public`     | `0`           | `1` if the judges list is publicly visible, `0` if not.                                                                                                                                                                                                                                                          |
 +---------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| `schools_public`    | `1`           | `1` if the schools are publicly visible on the public teams list.                                                                                                                                                                                                                                                |
++---------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| `debaters_public`   | `1`           | `1` if the debaters are publicly visible on the public teams list.                                                                                                                                                                                                                                                |
++---------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| `team_codes_backend`| `0`           | `1` if the team codes are used for all backend views, `0` if not.                                                                                                                                                                                                                                                |
```

Comment:
This phrasing probably isn't something most tab directors would understand. Maybe change to (`if team codes are used in non-public views, in addition to the public ones`)

Reply:
Check it out now, I changed backedn to non-public.  I didn't add the in addition clause as technically (?) one could only use team codes on the backend using these flags (not that they'd want to).